tests/tests/buffer.rs

```diff
@@ -326,3 +326,62 @@ static MINIMUM_BUFFER_BINDING_SIZE_DISPATCH: GpuTestConfiguration = GpuTestConfi
             let _ = encoder.finish();
         });
     });
+
+#[gpu_test]
+static CLEAR_OFFSET_OUTSIDE_RESOURCE_BOUNDS: GpuTestConfiguration = GpuTestConfiguration::new()
+    .parameters(TestParameters::default().test_features_limits())
```

Comment:
We don't need test_features_limits() here, just default

Reply:
Ah, okay. Fixed with 2e6138e169a2b1fa718402f4cd88bb3d57701c76 and 17c8f32cad0a1ce34ac2e978b5124fc9a1915775.